src/implementation/Client/GRPCClient/actor.ts

```diff
@@ -30,10 +30,10 @@ export default class GRPCClientActor implements IClientActorBuilder {
   create<T>(actorTypeClass: Class<T>): T {
     const builder = new ActorProxyBuilder<T>(
       actorTypeClass,
-      this.client.getClientHost(),
-      this.client.getClientPort(),
-      this.client.getClientCommunicationProtocol(),
-      this.client.getOptions(),
+      this.client.options.daprHost,
```

Comment:
is `client.options.` always set?

Reply:
Yup, if you look at GRPCClient or HTTPClient ctor, it always takes an option and sets it. The DaprClient class takes a partial option, and creates the entire object using defaults.